src/test/java/com/epam/tc/hw1/TestOfAddition.java

```diff
@@ -1,4 +1,4 @@
-package com.epam.tc.hw1;
+/*package com.epam.tc.hw1;
```

Comment:
Why did you comment out all of your first hw?

Reply:
Maven informed me about unstable work due to the calculator file and I decided to comment out the first hw

Reply:
I fixed